test/mock/core/api/transport/jrpc_processor_stub.hpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_JRPC_PROCESSOR_STUB_HPP
+#define KAGOME_JRPC_PROCESSOR_STUB_HPP
+
+#include <api/jrpc/value_converter.hpp>
+#include <boost/noncopyable.hpp>
+#include <mutex>
+
+#include "api/jrpc/jrpc_processor.hpp"
+#include "api/jrpc/jrpc_server_impl.hpp"
+#include "api/state/state_api.hpp"
+
+namespace kagome::api {
+
+  class JrpcProcessorStub : public JRpcProcessor, private boost::noncopyable {
```

Comment:
Hm, are you sure it won't be better to make it a mock? It'd be more flexible for use in tests.

Reply:
No, I think about state (results) testing there more right than test of behavior.